index.html

```diff
@@ -8,144 +8,227 @@
+            <input type="text" id="dob" name="dob" placeholder="06.19.2003" />
+          </fieldset> -->
+        <!-- </form> -->
+
+        <label for="Favorite Drake Song?" name="Favorite Drake Song?"
+          >Favorite Drake Song?</label
+        >
+        <select name="Favorite Drake Song?">
+          <option value="placeholder" selected></option>
+          <option value="Passionfruit">Passionfruit</option>
+          <option value="God's plan">God's plan</option>
+          <option value="Money in the Grave">Money in the Grave</option>
+          <option value="Hold, on we're going home">
+            Hold on we're going home
+          </option>
+        </select>
+      </div>
+      <!-- Testing -->
+    </div>
     <div>
-      <h2 id="connectHeading">Connect</h2>
+      <section>
+        <h2><legend>Leave a Message</legend></h2>
+        <form id="leave_message" name="leave_message">
+          <fieldset>
+            <div>
+              <label for="usersName">Name:</label>
+              <input
+                type="text"
+                name="usersName"
+                required="true"
+                placeholder="Cristian"
+              />
+            </div>
+            <br />
+            <div>
+              <label for="email">Email:</label>
+              <input
+                type="email"
+                name="email"
+                required="true"
+                placeholder="you@gmail.com"
+              />
+            </div>
+            <br />
+            <div>
+              <label for="usersMessage">Message:</label>
+              <textarea name="usersMessage" required="true"></textarea>
+            </div>
+            <br />
+            <div>
+              <button type="submit">Submit</button>
+              <!-- <button type="reset" id="resetButton">Reset</button> -->
+            </div>
+          </fieldset>
+        </form>
+      </section>
     </div>
-
     <div>
-      <p>
-        <a href="mailto: cmoran1654@gmail.com" target="_blank"
-          ><img
-            src="https://th.bing.com/th/id/OIP.qGd92IGgx7bJwhvx26xxIQHaHa?w=183&h=183&c=7&r=0&o=5&pid=1.7"
-            width="30"
-            height="25"
-        /></a>
-        <a
-          href="https://www.linkedin.com/in/cristian-moran-1b06841ba/"
-          target="_blank"
-          ><img
-            src="https://th.bing.com/th/id/OIP.q9s2sSjwz87hMp3bE2WpmwHaHa?w=184&h=184&c=7&r=0&o=5&pid=1.7"
-            width="30"
-            height="25"
-        /></a>
-
-        <a href="https://github.com/CristianMoran1 " target="_blank"
-          ><img
-            src="https://1000logos.net/wp-content/uploads/2021/05/GitHub-logo.png"
-            width="50"
-            height="30"
-        /></a>
-      </p>
+      <section id="messages">
+        <h2>Messages</h2>
+        <div>
+          <ul></ul>
+        </div>
+      </section>
     </div>
-    <footer id = "footer">
-    </footer>
+    <footer id="footer"></footer>
```

Comment:
One cool thing about HTML tags is that when you don't have anything inside of them, you can use a shortcut to close and open them at the same time: 

`<div/>` is the same as <div></div>`